src/components/NcEmptyContent/NcEmptyContent.vue

```diff
@@ -123,8 +154,11 @@ export default {
 		hasTitle() {
 			return this.title !== ''
 		},
+		/**
+		 * Check if a description is given as either property or slot
+		 */
 		hasDescription() {
-			return this.description !== ''
+			return this.description !== '' || this.$slots.description?.[0]
 		},
```

Comment:
Note: this is not well compatible with Vue 3 (by design). Isn't `this.$slots.description` enough to check if the description slot was passed?

Reply:
As stable7 will not see vue3 (I guess) we should discuss this for the version on the master branch.

Nevertheless this is required to keep the behavior the same, as an empty slot = empty property should not show the description nodes.

Reply:
> Nevertheless this is required to keep the behavior the same, as an empty slot = empty property should not show the description nodes.

By empty slot do you mean this usage:

```html
<NcEmptyContent>
<!-- nothing -->
</NcEmptyContent>
```

or this

```html
<NcEmptyContent>
  <template #description></template>
</NcEmptyContent>
```

?